service/src/test/java/com/commercetools/main/BasicPaymentRequestWorkflow.java

```diff
@@ -16,25 +16,44 @@
 import static com.commercetools.pspadapter.payone.domain.payone.PayonePostServiceImpl.executeGetRequest;
 import static java.lang.String.format;
 import static org.assertj.core.api.Assertions.assertThat;
+import static util.PaymentRequestHelperUtil.URL_HANDLE_PAYMENT;
+import static util.PaymentRequestHelperUtil.cleanupData;
+import static util.PaymentRequestHelperUtil.createPayment;
+import static util.PropertiesHelperUtil.getClientId;
+import static util.PropertiesHelperUtil.getClientSecret;
+import static util.PropertiesHelperUtil.getPayoneKey;
+import static util.PropertiesHelperUtil.getPayoneMerchantId;
+import static util.PropertiesHelperUtil.getPayonePortalId;
+import static util.PropertiesHelperUtil.getPayoneSubAccId;
+import static util.PropertiesHelperUtil.getProjectKey;
+import static util.PropertiesHelperUtil.getTenant;
 
 public class BasicPaymentRequestWorkflow {
 
-    private final String CT_PROJECT_KEY = "payone-integration-tests-heroku-17";
-    private final String CT_CLIENT_ID = "f4e4AIIfXh0zaC6k7sNkCI4Q";
-    private final String CT_CLIENT_SECRET = "zf_o_F0oknJ4ZOqOlhY4LiSmTOHCfX8h";
+    public static final int DEFAULT_PORT = 8080;
     private final Map<String, String> testInternalProperties = new HashMap<>();
-    private static final String URL_PATTERN = "http://localhost:%d/health";
 
     @Before
     public void setUp() throws URISyntaxException {
-        testInternalProperties.put("TENANTS", "TEST_DATA");
-        testInternalProperties.put("TEST_DATA_CT_PROJECT_KEY", CT_PROJECT_KEY);
-        testInternalProperties.put("TEST_DATA_CT_CLIENT_ID", CT_CLIENT_ID);
-        testInternalProperties.put("TEST_DATA_CT_CLIENT_SECRET", CT_CLIENT_SECRET);
-        testInternalProperties.put("TEST_DATA_PAYONE_KEY", "TvIOwFdSzOSVKE3Y");
-        testInternalProperties.put("TEST_DATA_PAYONE_MERCHANT_ID", "31102");
-        testInternalProperties.put("TEST_DATA_PAYONE_PORTAL_ID", "2022125");
-        testInternalProperties.put("TEST_DATA_PAYONE_SUBACC_ID", "31281");
+        cleanupData();
+        testInternalProperties.put("TENANTS", getTenant());
```

Comment:
Please also update the documentation how to run this project for devs so next time a new dev doesn't have to ask for info.

Reply:
https://github.com/commercetools/commercetools-payone-integration/pull/392/commits/b2aae19711c4b8ef87c605edee260d8a61dff561